.github/workflows/build-ci-container.yml

```diff
@@ -0,0 +1,51 @@
+
+name: Build CI Container
+
+permissions:
+  contents: read
+
+on:
```

Comment:
We probably want a `workflow_dispatch` here too for cases like rebuilding the container with the latest packages in the Ubuntu repos if there's something like a security update.

Reply:
I considered doing this, but decided not to, because I didn't want just anyone to be able to rebuild the container.  One option would be to add a workflow_dispatch, but with a user/team check.